gigascience/gigadb-website
# fix-2066: max length for attribute value set to 1000

# Pull request for issue: #2066

This is a pull request for the following functionalities:

max length for attribute value set to 1000

## How to test?
Go to https://gigadb.org/adminFile/update/id/546450
Scroll down to Attributes
Note that "embryophyta_odb10 C99.1%[S" was truncated during spreadsheet upload (a different issue).
Click Edit
Attempt to add "embryophyta_odb10 C:99.1%[S:77.9%,D:21.2%],F:0.4%,M:0.5%,n:1614"

## How have functionalities been implemented?

## Any issues with implementation?

## Any changes to automated tests?

AdminFileAttributes acceptance test

## Any changes to documentation?

## Any technical debt repayment?

## Any improvements to CI/CD pipeline?
